packages/@glimmer/application-test-helpers/src/app-builder.ts

```diff
@@ -18,9 +18,10 @@ import { buildAction, mainTemplate } from '@glimmer/application';
 import { compilable } from '@glimmer/opcode-compiler';
 import { Metadata } from '../../application/src/loaders/bytecode/loader';
 import { SimpleDocument } from '@simple-dom/interface';
-import { SSRApplication } from '@glimmer/ssr';
+import { SSRApplication, SET_INTERNAL_DYNAMIC_SCOPE } from '@glimmer/ssr';
```

Comment:
Given this is functionality we will probably want to expose for client-side rendering use cases as well, I'm thinking this symbol should be in `@glimmer/application` so it can be shared across all users of `BaseApplication`?

Reply:
Ah good point. I will move it over.